show/main.py

```diff
@@ -21,8 +21,15 @@
 from swsssdk import SonicV2Connector
 
 import mlnx
+from collections import OrderedDict
+from portconfig import get_child_ports
```

Comment:
This is a SONiC library import. Move above the line, `from swsssdk import ConfigDBConnector` (line 20)

Reply:
#done